d2l-button-shared-styles.html

```diff
@@ -1,7 +1,14 @@
 <link rel="import" href="../polymer/polymer.html">
 <style is="custom-style">
 	html {
+		--d2l-button-clear-focus: {
+			box-shadow: 0 0 0 4px rgba(0, 0, 0, 0);
+		};
+		--d2l-button-focus: {
+			box-shadow: 0 0 0 4px var(--d2l-color-celestine-light);
+		};
 		--d2l-button-spacing: 0.75rem;
+		--d2l-color-celestine-light: rgba(0, 111, 191, 0.3);
```

Comment:
OK good call on the other uses of it! Yeah even just moving the variable inside since it technically doesn't need to be a variable (yet) should help.